src/components/experiment-tracking/experiment-primary-toolbar/sliding-animation.js

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { Transition } from 'react-transition-group';
+
+const directions = {
+  leftToRight: {
+    entering: {
+      transform: 'translateX(0)',
+      visibility: 'visible',
+    },
+    entered: {
+      transform: 'translateX(0)',
+      visibility: 'visible',
+    },
+    exiting: {
+      transform: 'translateX(-34%)',
```

Comment:
I wonder if the `34%` value can also be configurable? If we want to use this again, what's the chance we'll want the same percentage? If too much to do right now, just leave as is and we can revisit later.

Reply:
Yes, we can do that by making that value to be dynamically changed. In this case we want to reposition the element horizontally from the start of the Sidebar (left and right) to the center, the Sidebar width is 56px and by 34% is animated to the center from both sides. Maybe we can set this value as we have `const duration = 300ms;` at the top of the component. What do you think?